src/components/form/input/input.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+    <div class="form-field form-field__input" :class="rootClasses">
+        <label>
+            <span class="form-field--title-label" v-if="!hasFloatingLabel && $slots.label">
+                <slot name="label"/>
+            </span>
+            <div class="form-field--input-container" :data-action-count="virtualTagsCount($slots.actions)">
```

Comment:
I would change the method name to `countVirtualTag` as it is a function and therefore is rather an action than a state.

Reply:
fixed with https://github.com/dreipol/vue-ui/pull/9/commits/ecec25284bf9a7c544b95fbd0c1c298d58dedcfe